services/web/server/requirements/_test.in

```diff
@@ -31,6 +31,8 @@ openapi-spec-validator
 python-dotenv
 redis
 tenacity
+aioboto3
```

Comment:
MINOR: keep alphabetical order please (can also group per topic and keep alphabetical order there, please